cookbooks/tomography_based_plate_motions/doc/tomography_based_plate_motions.md

```diff
@@ -65,3 +65,49 @@ The user can change several parameters to investigate the mantle forces and how
 - The reference viscosity profile.
 - The prescribed viscosity of the plate boundaries, which controls friction between plates.
 - To use viscous and neutrally buoyant cratons in the model.
+
+# Added complexity—Slab2 and initial topography
+For additional complexity, we use the Slab2 {cite}`hayes2018slab2` database that describes the three dimensional geometries of all the seismically active subduction zones in detail on Earth instead of the vertical slabs defined in the TM1 model. We also deform the outer shell by the initial topography. Incorporating these finer heterogeneities allow us to simulate Earth-like surface deformation in the models.
```

Comment:
```suggestion
For additional complexity, we use the Slab2 {cite}`hayes2018slab2` database that describes the three dimensional geometries of all the seismically active subduction zones in detail on Earth instead of the vertical slabs defined in the TM1 model. We also deform the outer shell by the initial topography. Incorporating these finer heterogeneities makes the simulations more physically realistic and may improve the fit to observed surface deformation patterns.
```